src/main/java/gov/adlnet/xapi/client/StatementClient.java

```diff
@@ -200,9 +200,7 @@ private StatementClient addFilter(String key, String value) {
 				client.filters = new TreeMap<String, String>();
 			}
 			if (this.filters != null) {
-				for (Entry<String, String> filter : filters.entrySet()) {
-					client.filters.put(filter.getKey(), filter.getValue());
-				}
+				client.filters.putAll(this.filters);
```

Comment:
No problem, totally understand. Just double checking I understand correctly though: are you also saying that you would not merge a patch which changes `private StatementClient addFilter(String key, String value) { ...` to `public StatementClient addFilter(String key, String value) { ...`?